indexer/ai/src/main/java/com/google/cloudsearch/AISkillDriver.java

```diff
@@ -0,0 +1,42 @@
+package com.google.cloudsearch;
+
+import com.google.cloudsearch.ai.AISkill;
+import com.google.cloudsearch.ai.AISkillSet;
+import com.google.cloudsearch.exceptions.InvalidConfigException;
+import com.google.common.collect.Multimap;
+import org.json.simple.JSONObject;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class AISkillDriver {
+    private AISkillSet skillSet;
+    private Boolean isInvalidConfig = false;
+    private static Logger log = Logger.getLogger(AISkillDriver.class.getName());
+
+    public AISkillDriver(JSONObject aiConfig, JSONObject schema) {
+        try{
```

Comment:
lets add a static method to this class "initialize(aiConfig, schemaConfig) and do all json parsing and skillset initialization in it.

Reply:
Sure, I will move it from the constructor to a different function.